.github/workflows/tests-outerloop.yml

```diff
@@ -172,4 +172,5 @@ jobs:
           path: |
             ${{ github.workspace }}/artifacts/log/*/*.binlog
             ${{ github.workspace }}/artifacts/log/*/TestLogs/**
+            ${{ github.workspace }}/artifacts/log/TestResults/*/*.trx
```

Comment:
This should be `artifacts/TestResults/**/*.trx`.

Reply:
Yes, thank you!